introduction.md

```diff
@@ -1,2 +1,3 @@
-I'm Branislav Petrovic and I want to introduce me to the dojo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?